apps/site/assets/ts/tnm/components/RouteCard.tsx

```diff
@@ -20,6 +24,25 @@ interface Props {
 const everyDirectionIsEmpty = (directions: Direction[]): boolean =>
   directions.every(directionIsEmpty);
 
+export const fetchAlerts = async (routeId: string): Promise<Alert[]> => {
+  const results = await fetchJsonOrThrow<Alert[]>(
+    `/api/alerts?route_ids=${routeId}`
+  );
+  if (isFetchFailed(results)) {
+    throw new Error(
+      `Failed to fetch Alert information: ${results.status} ${results.statusText}`
+    );
+  }
+
+  let alerts: Alert[] = [];
+  alerts = results.filter(isHighSeverityOrHighPriority) as Alert[];
+  return alerts;
```

Comment:
```suggestion
  return results.filter(isHighSeverityOrHighPriority);
```

This can be simplified!